README.md

```diff
@@ -76,6 +79,11 @@ class ExampleTest extends TestCase
      * @server APP_DEBUG=1
      * @putenv APP_HOST=localhost
      */
+    #[Env('APP_ENV', 'foo')]
+    #[Env('APP_DEBUG', '0')]
+    #[Server('APP_ENV', 'bar')]
+    #[Server('APP_DEBUG', '1')]
+    #[Putenv('APP_HOST', 'localhost')]
```

Comment:
Can we present two separate examples instead of mixing the two? One for attributes and one for annotations. Attributes can go first as annotations will eventually be removed.